manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,321 @@ def get_coordinate_labels(self, *numbers, **kwargs):
     def add_coordinates(self, *numbers):
         self.add(self.get_coordinate_labels(*numbers))
         return self
+
+
+class PolarPlane(NumberPlane):
+    """A version of `NumberPlane` for polar coordinates.
```

Comment:
If the docstring were a raw string (`r"""`), the backslashes in it would not have to be escaped. I would prefer that (but won't block the PR over it).